cohere/client.py

```diff
@@ -334,23 +318,20 @@ def summarize(
             additional_command (str): (Optional) Modifier for the underlying prompt, must \
                 complete the sentence "Generate a summary _".
 
-        Example:
-        ```
-        res = co.summarize(text="Stock market report for today...")
-        print(res.summary)
-        ```
-
-        Example:
-        ```
-        res = co.summarize(
-            text="Stock market report for today...",
-            model="summarize-xlarge",
-            length="long",
-            format="bullets",
-            temperature=0.9,
-            additional_command="focusing on the highest performing stocks")
-        print(res.summary)
-        ```
+        Examples:
+            Summarize a text:
+                >>> res = co.summarize(text="Stock market report for today...")
+                >>> print(res.summary)
+
+            Summarize a text with a specific model and prompt:
+                >>> res = co.summarize(
+                >>>     text="Stock market report for today...",
+                >>>     model="summarize-xlarge",
+                >>>     length="long",
+                >>>     format="bullets",
+                >>>     temperature=0.9,
```

Comment:
```suggestion
                >>>     temperature=0.3,
```
As per matthias's request